src/daemon/ca/events.rs

```diff
@@ -71,19 +74,195 @@ impl fmt::Display for IniDet {
 /// Describes an update to the set of ROAs under a ResourceClass.
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct RoaUpdates {
-    #[serde(skip_serializing_if = "HashMap::is_empty", default = "HashMap::new")]
+    #[serde(
+        skip_serializing_if = "HashMap::is_empty",
+        default = "HashMap::new",
+        with = "updated_map"
```

Comment:
What's the reason for using an own serializer over the default Serde HashMap serialization?